go.mod

```diff
@@ -4,8 +4,7 @@ go 1.16
 
 require (
 	github.com/container-storage-interface/spec v1.6.0
-	github.com/go-logr/logr v1.2.3 // indirect
-	github.com/golang/mock v1.5.0
+	github.com/golang/mock v1.6.0
 	github.com/kubernetes-csi/csi-lib-utils v0.11.0
 	github.com/kubernetes-csi/csi-test/v4 v4.0.0-20200806214950-555d70a11a8b
```

Comment:
Can you update this to v5.0.0?
https://github.com/kubernetes-csi/csi-test/releases/tag/v5.0.0

Reply:
@xing-yang done.. ptal.. thanks